### PR TITLE
push my task files

### PR DESCRIPTION
**Pull request by Perpee**
GROUP B
Please check my work @promise-J and @ChidiPatrick and @Pieta9221

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
\ No newline at end of file
+##Bettybingo-  https://perpe123.github.io/Lesson_3_task_5_Obiefuna_Toochi/
+
+##How to run/test this project
+
+##ensure you have git installed properly on your machine
+
+##ensure you have node js installed in your machine
+
+##open your terminal and run "git clone "
```
Comment (Pieta9221) [inline README.md]: Add repo link

File: css/style.css [hunk 2]
```diff
@@ -1,52 +1,330 @@
...
+
+header nav a {
+    margin-right: 20px;
+    font-size: 15px;
+    text-decoration: none;
+    color: wheat;
+  }
+  
+  header nav a:hover {
+    color: rgb(230 168 53);
```
Comment (Pieta9221) [inline css/style.css]: You might want to select a different hover color or implement a different hover behavior

File: css/style.css [hunk 3]
```diff
@@ -1,52 +1,330 @@
...
+  height: 300px;
+  background-color: wheat;
+  display: flex;
+  flex-direction: column;
+  padding: 10px;
+  transition: all 1s;
+}
+
+.content:hover {
+  transform: rotate(360deg);
```
Comment (Pieta9221) [inline css/style.css:206]: Something seems to be faulty here. The animation is not seamless and affects multiple items at the same time. Kindly fix this

**Review by Pieta9221 — COMMENTED** [review]
Good job :+1: :+1: :+1:
It can be better!
Kindly make the changes before merging

**Comment by Perpee** [comment]
Ok.. thanks

**Review by promise-J — APPROVED** [review]
@Perpe123. I am impressed at the neatness. However I am worried at the high quality of the images which has in turn contributed to the low performance of the web site. You might read up on lazy loafing of images to improve the performance. However, I feel it is out of the context for the review. You are free to merge.

**Review by ChidiPatrick — APPROVED** [review]
Nice job you've done

**Review by ChidiPatrick — APPROVED** [review]
Nice job you've done

**Pull request merged**